tsconfig.json

```diff
@@ -2,9 +2,8 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "compilerOptions": {
     "target": "ES2022",
-    "module": "NodeNext",
+    "module": "CommonJS",
```

Comment:
this was changed in order to work with [this mime package for node](https://github.com/broofa/mime)